vagrant/provisioning/arkcase-app-upgrade.yml

```diff
@@ -11,6 +11,8 @@
       tags: [core]
     - role: foia
       tags: [foia]
+    - role: privacy
```

Comment:
I guess we control whether foia or privacy is activated by using tags?

Reply:
Yes. We trigger the deployment with tags from the gitlab-ci.yaml file on every build.